real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyConfigFile.java

```diff
@@ -3,17 +3,21 @@
 import java.io.FileNotFoundException;
 import java.io.FileReader;
 import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.Arrays;
 
 import com.orsoncharts.util.json.JSONObject;
 import com.orsoncharts.util.json.parser.JSONParser;
 import com.orsoncharts.util.json.parser.ParseException;
 
 // where all system configuration values should be placed
-// add jason parser for config
+// add jason parser for RobobuggyConfigFile
 
-public class config {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1795141970/issues/source?bid=2938665&fileBranchId=3059924#l23)